modules/dnn/src/caffe/caffe_importer.cpp

```diff
@@ -388,6 +388,33 @@ class CaffeImporter
                     layerParams.blobs[1].setTo(1);  // std
                 }
             }
+            else if (type == "Axpy")
+            {
+                CV_Assert_N(layer.bottom_size() == 3, layer.top_size() == 1);
+
+                std::string scaleName = name + "/scale";
+                int repetitions = layerCounter[scaleName]++;
+                if (repetitions) {
+                    scaleName += String("_") + toString(repetitions);
+                }
+
+                LayerParams scaleParams;
+                scaleParams.set("axis", 1);
+                scaleParams.set("has_bias", false);
+                int scaleId = dstNet.addLayer(scaleName, "Scale", scaleParams);
+                addInput(layer.bottom(2), scaleId, 0, dstNet);
+                addInput(layer.bottom(0), scaleId, 1, dstNet);
+                addOutput(layer, scaleId, 0);
+                net.mutable_layer(li)->set_bottom(0, layer.top(0));
+
+                LayerParams eltwiseParams;
+                eltwiseParams.set("operation", "sum");
+                int eltwiseId = dstNet.addLayer(name, "Eltwise", eltwiseParams);
+                addInput(layer.top(0), eltwiseId, 0, dstNet);
+                addInput(layer.bottom(1), eltwiseId, 1, dstNet);
+                addOutput(layer, eltwiseId, 0);
+                continue;
```

Comment:
If we replace Eltwise layer creation to just `type = "Eltwise"` (like in the next case), will it work?